README.md

```diff
@@ -2,6 +2,14 @@
 
 In order to use this library, you need to have an account on <https://pusher.com/channels>. After registering, you will need the application credentials for your app.
 
+## Supported platforms
+
+This SDK supports **Node.js** version 4 or newer.
```

Comment:
If we follow [the NodeJS version support](https://nodejs.org/en/about/releases/), I would say only 6+. (But we can still keep tests for older versions, if you wish.)

Reply:
Sounds good to me.